src/tests/App.test.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
```

Comment:
When trying to run the tests theres a 'Cannot use import statement outside of a module' error coming up, this is because of the version of axios youre using, they should fix it at some point but until then try adding this to your package.json:

```
"jest": {
  "transformIgnorePatterns": ["node_modules/(?!axios)/"]
}
```

Or update the “test” line in your package.json to this:

```
"test": "react-scripts test --transformIgnorePatterns \"node_modules/(?!axios)/\""
```

To be clear, this is nothing youre doing wrong, just something that needs to be sorted with axios